AutomatedTesting/Gem/PythonTests/scripting/ScriptEvent_AddRemoveMethod_UpdatesInSC.py

```diff
@@ -38,161 +20,65 @@ class TestScriptEvent_AddRemoveMethod_UpdatesInSC():
-            NUM_TEST_METHODS * WAIT_TIME_3
-        )
-        Report.result(Tests.child_2_created, result)
-
-        # 5) Update MethodNames and save file, (update all Method names to make it easier to search in SC later)
-        # Expand the EventName initially
-        self.expand_container_rows(DEFAULT_SCRIPT_EVENT)
-        # Expand Name fields under it
-        self.expand_container_rows("Name")
-        count = 0  # 2 Method names will be updated Ex: test_method_name_0, test_method_name_1
-        container = self.asset_editor_widget.findChild(QtWidgets.QWidget, "ContainerForRows")
-        children = container.findChildren(QtWidgets.QFrame, "Name")
-        for child in children:
-            line_edit = child.findChild(QtWidgets.QLineEdit)
-            if line_edit and line_edit.text() == DEFAULT_METHOD_NAME:
-                line_edit.setText(f"{NODE_TEST_METHOD}_{count}")
-                count += 1
-        self.save_file()
-
-        # 6) Verify if the new node exist in SC (search in node palette)
-        general.open_pane(SCRIPT_CANVAS_UI)
-        helper.wait_for_condition(lambda: general.is_pane_visible(SCRIPT_CANVAS_UI), WAIT_TIME_3)
-        self.initialize_sc_qt_objects()
-        self.node_palette_search(f"{NODE_TEST_METHOD}_1")
-        method_result_1 = helper.wait_for_condition(
-            lambda: pyside_utils.find_child_by_pattern(self.node_palette_tree, {"text": f"{NODE_TEST_METHOD}_1"}) is not None, WAIT_TIME_3)
-        Report.result(Tests.method_added, method_result_1)
-
-        # 7) Delete one method and save
-        self.initialize_asset_editor_qt_objects()
-        for child in container.findChildren(QtWidgets.QFrame, DEFAULT_SCRIPT_EVENT):
-            if child.findChild(QtWidgets.QToolButton, ""):
-                child.findChild(QtWidgets.QToolButton, "").click()
-                break
-        self.save_file()
-
-        # 8) Verify if the node is removed in SC (search in node palette)
-        self.initialize_sc_qt_objects()
-        self.node_palette_search(f"{NODE_TEST_METHOD}_0")
-        method_result_0 = helper.wait_for_condition(
-            lambda: pyside_utils.find_child_by_pattern(self.node_palette_tree, {"text": f"{NODE_TEST_METHOD}_0"}) is None, WAIT_TIME_3)
-        Report.result(Tests.method_removed, method_result_0)
-
-        # 9) Close Asset Editor
-        general.close_pane(ASSET_EDITOR_UI)
-        general.close_pane(SCRIPT_CANVAS_UI)
-
-
-test = TestScriptEvent_AddRemoveMethod_UpdatesInSC()
-test.run_test()
+    # Preconditions
+    import os
+    import azlmbr.paths as paths
+    import editor_python_test_tools.script_canvas_tools_qt as sc_tools_qt
+    from editor_python_test_tools.utils import Report
+    import azlmbr.legacy.general as general
+    from consts.asset_editor import (SCRIPT_EVENT_UI, NODE_TEST_METHOD)
+
+    general.idle_enable(True)
+
+    # 1) Initialize Qt Test objects and open Asset Editor
+    qtpy_o3de_editor = sc_tools_qt.qtpy_o3de_editor
```

Comment:
Why don't we just do the following import:

`from editor_python_test_tools.QtPyO3DEEditor import QtPyO3DEEditor`

So we can then do this:

`qtpy_o3de_editor = QtPyO3DEEditor()`

We should be doing our best to deprecate `script_canvas_tools_qt.py` to be replaced by the libraries where possible.

In fact, anything that was abstracted out of that file into the QtPy libs for this PR into the libs should be removed from this file.

Reply:
updated this. I will need to go back and change the way we get a handle on the QtPy object in the previous test script as well.